src/v/storage/tests/compaction_e2e_test.cc

```diff
@@ -725,8 +725,6 @@ TEST_F(CompactionFixtureTest, TestTombstones) {
     // Generate a tombstone record for "key0".
```

Comment:
> storage: modify sleep timers in compaction_e2e_test.cc
> To avoid race-y failures.

Maybe it's just me, but I don't think anyone looking at this commit in the future would be able to understand what was fixed or what the problem was to begin with.